docs/running/notebooks/source_files/program_2.py

```diff
@@ -1,22 +1,31 @@
-# source_files/program_2.py
+# source_files/program_4.py
```

Comment:
the file name and the comment are different now... do we also need to rename the file?

Reply:
The filenames have been renamed. These comments at the top of the file seem to be the one thing I missed, thanks

Reply:
cool ... seems like there was some git weirdness (wouldn't be the first time...) as usually it shows when files are renamed...

Reply:
I know, I can't figure out why it isn't at least showing the arrow from old to new name in the diff... hmm ... Going to 100% verify they've been renamed from what they are in `main`, but I'm fairly confident

Reply:
it may just be git "decided" to pick up on the line changes rather than the rename... I've seen it do stranger things